bin/broker_submission.py

```diff
@@ -20,28 +20,40 @@
 from ebi_eva_common_pyutils.logger import logging_config as log_cfg
 
 from eva_submission.eload_brokering import EloadBrokering
+from eva_submission.eload_utils import check_existing_project
 from eva_submission.submission_config import load_config
 
 logger = log_cfg.get_logger(__name__)
 
 
+def ENA_Project(project):
+    # Helper class to validate early that the project provided exist in ENA and is public
+    if not check_existing_project(str(project)):
+        logger.warning(f'Project {project} provided does not exist in ENA.')
+        raise ValueError
```

Comment:
Should probably have an error message...

Reply:
See above